Source/Diagnostics/ComputeDiagFunctors/ParticleReductionFunctor.cpp

```diff
@@ -26,8 +26,8 @@ ParticleReductionFunctor::ParticleReductionFunctor (const amrex::MultiFab* mf_sr
 
     // Allocate and compile a parser based on the input string fn_str
     m_map_fn_parser = std::make_unique<amrex::Parser>(makeParser(
-                fn_str, {"x", "y", "z", "ux", "uy", "uz"}));
-    m_map_fn = m_map_fn_parser->compile<6>();
+                fn_str, {"x", "y", "z", "ux", "uy", "uz", "us"}));
```

Comment:
```suggestion
                fn_str, {"x", "y", "z", "ux", "uy", "uz", "upstream"}));
```

Reply:
"us" after ux, uy, uz can be confused with some new momentum component
Better to write this out as upstream